packages/tracing/src/browser/metrics.ts

```diff
@@ -179,6 +214,63 @@ export class MetricsInstrumentation {
       // Do nothing if the browser doesn't support this API.
     }
   }
+
+  /** Starts tracking the First Input Delay on the current page. */
+  private _trackFID(): void {
+    // Based on reference implementation from https://web.dev/fid/#measure-fid-in-javascript.
+    // Use a try/catch instead of feature detecting `first-input`
+    // support, since some browsers throw when using the new `type` option.
```

Comment:
This proved to be not enough for LCP, we need to pay special attention to code running on callbacks :)

Reply:
I added some additional feature detection here https://github.com/getsentry/sentry-javascript/pull/2909/commits/8c81de50fd5415fd650cb42b039e417821d13b4a